packages/paste-core/components/chat-log/__tests__/index.spec.tsx

```diff
@@ -22,6 +32,20 @@ describe('ChatBubble', () => {
   });
 });
 
+describe('ChatAttachment', () => {
+  it('should render an icon, anchor, and text', () => {
+    const {container} = render(
+      <ChatAttachment attachmentIcon={DownloadIcon}>
+        <ChatAttachmentLink href="www.google.com">document</ChatAttachmentLink>
+        <ChatAttachmentDescription>description</ChatAttachmentDescription>
+      </ChatAttachment>
+    );
+    expect(container.querySelector('[data-paste-element="ICON"]')).toBeDefined();
+    expect(screen.getByRole('link')).toBeDefined();
+    expect(screen.getByText('description')).toBeDefined();
+  });
+});
+
```

Comment:
+1. It think it makes sense to have a separate a11y test for an attachment in the attachmentContainer component because its not meant to be used in a chat log.